core/api.cfc

```diff
@@ -852,7 +852,7 @@
 	<cffunction name="sortURIMatchOrder" access="private" output="false">
 		<cfargument name="endpoints" />
 		<cfset var URIMatchOrder = listToArray( structKeyList(arguments.endpoints, chr(10)), chr(10) ) />
-		<cfset arraySort(URIMatchOrder, "text", "desc") />
+		<cfset arraySort(URIMatchOrder, "textnocase", "desc") />
```

Comment:
This line change gave me quite a bit of pause. I was worried that we might lose some nuance of the way we handle incoming URI's and some tiny corner case bug might be introduced by moving from more explicit to less...

But then I realized that we're using these strings as patterns in a `reMatchNoCase()` so it was already case insensitive.

https://github.com/atuttle/Taffy/blob/a24e0f8f9a98c02d8d044a2a341adbcc1e741fb5/core/api.cfc#L864

I am still mildly curious what problem this solves. The only thing I can think of is carelessness from developers in using consistent casing that might result in separation of resources in the dashboard that might have otherwise been grouped:

For example, using a case-sensitive sort:

    /Users/abb
    /users/bbb
    /users/aaa

Whereas a case-insensitive sort would result in:

    /users/bbb
    /Users/abb
    /users/aaa